listenbrainz/webserver/views/explore.py

```diff
@@ -56,10 +57,27 @@ def fresh_releases():
 def artist_similarity():
     """ Explore artist similarity """
 
-    return render_template(
-        "explore/artist-similarity.html",
-        props=orjson.dumps({}).decode("utf-8")
-    )
+    with psycopg2.connect(current_app.config["SQLALCHEMY_TIMESCALE_URI"]) as ts_conn, \
+            ts_conn.cursor() as ts_curs:
+        ts_curs.execute("""
+                        SELECT artist_mbid::TEXT
+                            FROM popularity.artist
+                            ORDER BY total_listen_count DESC
+                            LIMIT 1
```

Comment:
Here I'm fetching the top artist. Alternatively we can fetch the sitewide trending artist for this month / this week. Which option would be the best here? @MonkeyDo